projects/table-test/src/app/app.component.ts

```diff
@@ -14,18 +14,18 @@ export class AppComponent {
 			active: false,
 			id: 'id1',
 		},
-		{
-			name: 'Tools',
-			firstName: 'Ngx',
-			active: true,
-			id: 'id3',
-		},
-		{
-			name: 'BCD',
-			firstName: 'A',
-			active: true,
-			id: 'id2',
-		},
+		// {
+		// 	name: 'Tools',
+		// 	firstName: 'Ngx',
+		// 	active: true,
+		// 	id: 'id3',
+		// },
+		// {
+		// 	name: 'BCD',
+		// 	firstName: 'A',
+		// 	active: true,
+		// 	id: 'id2',
+		// },
```

Comment:
Waarom zijn deze gecomment en niet verwijderd?

Reply:
Puur een test applicatie dus het maakt eigenlijk niet echt uit, als we dan morgen toch weer met meerdere items willen werken kunnen we dat weer ontcommenten.